cmd/builder/README.md

```diff
@@ -10,12 +10,10 @@ $ GO111MODULE=on go install go.opentelemetry.io/collector/cmd/builder@latest
 $ cat > otelcol-builder.yaml <<EOF
 exporters:
   - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.40.0"
-  - import: go.opentelemetry.io/collector/exporter/loggingexporter
-    gomod: go.opentelemetry.io/collector v0.40.0
+  - gomod: go.opentelemetry.io/collector/exporter/loggingexporter v0.63.0
 
 receivers:
-  - import: go.opentelemetry.io/collector/receiver/otlpreceiver
-    gomod: go.opentelemetry.io/collector v0.40.0
+  - gomod: go.opentelemetry.io/collector/receiver/otlpreceiver v0.64.0
```

Comment:
Update all versions here?

Reply:
Updated all versions except batch processor, that's coming in a follow up PR